src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE

```diff
@@ -208,9 +208,9 @@ http_archive(
 http_archive(
     name = "remote_java_tools",
     build_file = "@bazel_tools//tools/jdk:BUILD.pkg",
-    sha256 = "bcfc1a3dd0d638a49fed50f17f0f1f7d77101debf19ae2c82119c82459a9a8d1",
+    #sha256 = "e8a47983fbd1f6e8b8d42bfbde9fde7652534c331edd272e27707779f61ae54f",
```

Comment:
Why did you comment this out?

Reply:
Technically the PR was not ready for review yet, I only asked for an opinion in the comments section. :) It was commented so I can test the PR with an intermediate archive. It points now to the latest release.